contracts/staking/IStaking.sol

```diff
@@ -107,25 +113,10 @@ interface IStaking is IStakingData {
 
     function closeAllocation(address _allocationID, bytes32 _poi) external;
 
-    function closeAllocationMany(CloseAllocationRequest[] calldata _requests) external;
```

Comment:
whats the significance on us removing these functions on the Indexer Agent? Do you have to write custom code in the agent to use mutlicall for :

- closeAllocationMany()
- closeAndAllocate()
- claimMany()

Reply:
The agent does not use any of those functions yet. Anyone calling those is because they are using something custom. Whoever is using that interface will need to use multicall, any upgrade will need to be communicated with time.

Reply:
okay that is good for me!

maybe we could keep the functions if it were a smaller contract. but staking is so big - it will be useful to delete these for future upgrades